### DISCUSSION TITLE
Studio: SQL vs SQL Script

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#541] [Q&A] [answered]
Dear Community,

can somebody give me a hint on the difference between `SQL` and `SQL Script` in the Studio?

Thank You

![sql_vs_sql-script](https://user-images.githubusercontent.com/4374619/193400085-ad9f42fa-ba58-492b-b8ae-552567dd692b.png)

**Comment by lvca** [comment, marked as answer]
The SQL script allows multiple statements (separated with `;`). Also, it allows using variables (LET statement) and more. Check that out at: https://docs.arcadedb.com/#_sql-batch.
